frontend-react/src/components/Table/TestTable.tsx

```diff
@@ -23,6 +23,11 @@ const dummyRowTwo = {
     five: "transform this",
 };
 
+// Exported for test purposes
+export const sampleCallback = () => {
+    console.log("Callback works!");
+};
+
 /* This component is specifically configured to help test the
  * Table component. Any  */
 export const TestTable = () => {
```

Comment:
I don't want to have to use this component for Table tests; will create a tech debt item to refactor this out into functions used to create whatever mock data we need to set up the tables for tests.

Reply:
Sounds good. 

Search for `export const _exportForTesting`. Not sure it's useful here but it's an attempt to clearly define exports that are just used for unit tests.

Reply:
I've extracted some easy generative logic in PR #5513 to begin the process of weening off of TestTable! They utils make the most sense in the Table.test file and I'll probably export them from there if needed 👍🏼